requirements.txt

```diff
@@ -1,11 +1,11 @@
-numpy
-pandas
-galsim
-nose
-scikit-learn
-astropy
-matplotlib
+pybind11
+numpy>=1.13
+future>=0.15
+matplotlib>=2.2.2
+astropy==2.0
+pandas>=0.20
+scikit-learn==0.19.2
 corner
-om10
-scipy
-git+git://github.com/drphilmarshall/lenspop.git#egg=lenspop
+git+git://github.com/jiwoncpark/lenspop.git@master#egg=lenspop
+git+git://github.com/jiwoncpark/om10.git@master#egg=om10
+galsim
```

Comment:
Nice requirements file! Does galsim not need a specific version number? Maybe not.

Reply:
On a second thought, I think it'd be good to fix the GalSim installation to version 2.0, in case there are future updates that break the current implementation. This is unlikely because we only use very basic GalSim functions but good to be defensive!